dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -240,17 +243,13 @@ export function AppRepoForm(props: IAppRepoFormProps) {
   };
 
   return (
-    <form onSubmit={handleInstallClick} className="app-repo-form">
-      <div className="clr-form-control">
-        <label htmlFor="kubeapps-repo-name" className="clr-control-label">
-          Name
-        </label>
-        <div className="clr-control-container">
-          <div className="clr-input-wrapper">
+    <>
```

Comment:
Out of interest, why is this required for the migration, since `form` below is still the only single element?

Reply:
It's not required (I was just testing things).